Gemfile

```diff
@@ -69,4 +69,4 @@ group :test do
 end
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+gem 'tzinfo-data'
```

Comment:
Does this need to be changed? The gem is only required for Windows.

Reply:
This was causing an error during the Docker build if it was left as it was